code/modules/antagonists/heretic/moon_lunatic.dm

```diff
@@ -0,0 +1,51 @@
+// A type of antagonist created by the moon ascension
+/datum/antagonist/lunatic
+	name = "\improper Lunatic"
+	hijack_speed = 0
+	antagpanel_category = ANTAG_GROUP_HORRORS
+	show_in_antagpanel = FALSE
+	suicide_cry = "PRAISE THE RINGLEADER!!"
+	antag_moodlet = /datum/mood_event/heretics/lunatic
+	can_assign_self_objectives = FALSE
+	hardcore_random_bonus = FALSE
+	// The mind of the ascended heretic who created us
+	var/datum/mind/ascended_heretic
+	// The body of the ascended heretic who created us
+	var/mob/living/carbon/human/ascended_body
+
+
+/// Runs when the moon heretic creates us, used to give the lunatic a master
+/datum/antagonist/lunatic/proc/set_master(datum/mind/heretic_master, mob/living/carbon/human/heretic_body)
+	src.ascended_heretic = heretic_master
+	src.ascended_body = heretic_body
+
+	var/datum/objective/lunatic_obj = new()
+	lunatic_obj.explanation_text = "Assist your master [heretic_master]."
+
+	to_chat(owner, span_boldnotice("Ruin the lie, save the truth through obeying [heretic_master] the ringleader!"))
+
+/datum/antagonist/lunatic/greet()
+	owner.current.playsound_local(get_turf(owner.current), 'sound/effects/moon_parade.ogg', 100, FALSE, pressure_affected = FALSE, use_reverb = FALSE)//subject to change
```

Comment:
this comment will never be read, if you want it to be move it up to the line before (or multiline this proc)